src/data/sidebar.ts

```diff
@@ -0,0 +1,56 @@
+import { prisma } from "data/prisma";
+
+async function getLocationLinksData() {
+  const results = await prisma.location.findMany({
+    include: {
+      members: true,
+    },
+  });
```

Comment:
I lost a _lot_ of time with a bug here. I was trying to do a `select` statement and select only what was needed from the child relation (members), but I was getting this prisma panic. Only way to get around it was just to include the whole child. I had similar logic in other places that this didn't happen to so idk what was up.